WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -324,8 +335,39 @@ public void loggedInViaUsernamePassword(ArrayList<Integer> oldSitesIds) {
     }
 
     @Override
-    public void help() {
-        ToastUtils.showToast(this, "Help is not implemented yet.");
+    public void helpEmailScreen(String email) {
+        launchHelpshift(null, email,
+                HelpshiftHelper.chooseHelpshiftLoginTag(getLoginMode() == LoginMode.JETPACK_STATS, true));
```

Comment:
Hmm, there shouldn't be any other case beyond those 3. When in the email screen, we are either in plain wpcom mode or in Jetpack mode and setting the `isWPComMode` parameter to `true` helps `chooseHelpshiftLoginTag` determine the proper tag. `chooseHelpshiftLoginTag` could be refactored to just use the LoginMode directly but that will need to happen after the old login classes get removed (to avoid having them depend on LoginMode as well).